scripts/Dockerfile.sample

```diff
@@ -0,0 +1,43 @@
+FROM <distro-specific-base-image>
```

Comment:
I'm not sure what benefit we get from having this Dockerfile in here. In fact, if it's just a sample and not meant to be runnable directly and used via any kind of CI or anything I can see it getting stagnant over time.

Reply:
Do u think it would help if we add distro specific files under a new scripts/distro folder and have that add over time.

```
distro/rh/Dockerfile
distro/centos/Dockerfile 
```

Reply:
I think container images are better suited for downstream maintainers to handle.  We don't intend to provide an upstream golang-fips container image, so we don't need to include a Dockerfile.